ops/infrastructure/roles/centos-eol-fix/tasks/main.yml

```diff
@@ -17,7 +17,7 @@
 - name:
   ansible.builtin.rpm_key:
     state: present
-    key: https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG
+    key: https://download.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-RHEL
```

Comment:
This code change in bastion script is not required anymore because GPG key repo problem has already been fixed.

Reply:
After rebasing with the latest commits from the `upstream/develop`, the changes have been updated.